types/fontname/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for fontname 1.0
+// Project: https://github.com/danbovey/fontname
+// Definitions by: Arystan Kaliakparov <https://github.com/arys>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface FontInfo {
+  copyright?: string;
+  fontFamily?: string;
+  fontSubfamily?: string;
+  ID?: string;
+  fullName?: string;
+  version?: string;
+  postScriptName?: string;
+  trademark?: string;
+  manufacturer?: string;
+  designer?: string;
+  description?: string;
+  urlVendor?: string;
+  urlDesigner?: string;
+  licence?: string;
+  licenceURL?: string;
+  typoFamilyName?: string;
+  typoSubfamilyName?: string;
+  compatibleFull?: string;
+  sampleText?: string;
+  postScriptCID?: string;
+  wwsFamilyName?: string;
+  wwsSubfamilyName?: string;
+  lightPalette?: string;
+  darkPalette?: string;
+  preferredFamily?: string;
+  preferredSubfamily?: string;
+}
+
+declare class FontName {
+  parse(buffer: ArrayBuffer): FontInfo[];
+}
+declare const _default: FontName;
+// tslint:disable-next-line:npm-naming
+export default _default;
```

Comment:
this one looks like old, good UMD module:
https://github.com/danbovey/fontname/blob/master/index.js#L1
so this applies:
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library

so:
```ts
export interface FontInfo {
    ...
}
export function parse(buffer: ArrayBuffer): FontInfo[];

export as namespace FontName;
```
```ts
import FontName = require('fontname');

FontName.parse(new ArrayBuffer(8));
```